File: 백준/Bronze/17598. Animal King Election/Animal King Election.py
import sys
input=lambda:sys.stdin.readline().rstrip()
cntL,cntT=0,0
for _ in range(9):
    t=input()
    if t=="Lion":
        cntL+=1
    else:
        cntT+=1
if cntT>=5:
    print("Tiger")
elif cntL>=5:
    print("Lion")

File: 백준/Bronze/17598. Animal King Election/Animal King Election.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int main(void) {
    int cntL = 0, cntT = 0;
    char *t = (char *)malloc(sizeof(char) * 6);
    for (int i = 0; i < 9; i++) {
        scanf("%s", t);
        if (strcmp(t, "Lion") == 0)
            cntL += 1;
        else
            cntT += 1;
    }
    if (cntL >= 5)
        printf("%s", "Lion");
    else if (cntT >= 5)
        printf("%s", "Tiger");
    return 0;
}